### ISSUE TITLE
duplicate release entry in appdata

### ISSUE THREAD

**Issue by bhepple** [babluboy/bookworm#306]
The file **data/com.github.babluboy.bookworm.appdata.xml.in** has a duplicate release entry for 1.0.0

The first one should be deleted so that the releases appear in correct order.

This duplicate entry upsets the fedora validation utility:

```
appstream-util validate-relax --nonet /home/bhepple/rpmbuild/BUILDROOT/bookworm-1.1.2-3.fc31.x86_64/usr/share/metainfo/com.github.babluboy.bookworm.appdata.xml
/home/bhepple/rpmbuild/BUILDROOT/bookworm-1.1.2-3.fc31.x86_64/usr/share/metainfo/com.github.babluboy.bookworm.appdata.xml: FAILED:
• tag-invalid           : <release> version was duplicated
• tag-invalid           : <release> versions are not in order [1.0.0 before 1.1.0]
Validation of files failed
```

**Comment by babluboy** [comment]
@bhepple Thanks for raising this. It might have crept in when I was playing around with appdata.xml. I think I fixed this and now I have the file [appdata.xml](https://github.com/babluboy/bookworm/blob/master/data/com.github.babluboy.bookworm.appdata.xml.in).in which is converted into appdata.xml during the build process. Hopefully the latest code in 1.1.2 does not have this issue.

**Comment by bhepple** [comment]
Hi Siddhartha,

I'm working in 1.1.2 assuming that it's a release and therefore unchanging!! Hope that's the case otherwise it'll mess up my packaging for fedora.

For this 1.1.2 release I have implemented a diff patch to correct the error and sneak it past fedora's validation.

Cheers

Bob

**Comment by babluboy** [comment]
@bhepple This has been resolved by pull request #304. Thanks to @eclipseo

**Comment by babluboy** [comment]
@bhepple So while you can progress the release for 1.1.2 with patching, the issue will not be there for the next release 1.1.3
